test/smoke-limbo/no-loop-3/Makefile

```diff
@@ -8,7 +8,7 @@ RUNENV      += LIBOMPTARGET_KERNEL_TRACE=1
 
 RUNCMD      = ./$(TESTNAME) 2>&1 | $(FILECHECK) $(TESTSRC_MAIN)
 
-CFLAGS       += -Ofast
+CFLAGS       += -O3 -fopenmp-target-fast
```

Comment:
Question which does apply to a lot of other changes:
When should `-fopenmp-target-fast` be used (in conjunction with `-O3`) and when to omit?